frontend/src/metabase/home/homepage/components/HomeCard/HomeCard.styled.tsx

```diff
@@ -1,6 +1,7 @@
 import styled from "@emotion/styled";
 import { alpha, color } from "metabase/lib/colors";
 import Link from "metabase/core/components/Link";
+import { breakpointMinExtraLarge } from "metabase/styled-components/theme";
```

Comment:
I'm not sure if it's `breakpointMinExtraLarge` or just `breakpointMinLarge`

Reply:
I tried both and found myself preferring `breakpointMinLarge`, so I've pushed a commit that makes that change.